test/unit/profile_spec.lua

```diff
@@ -110,20 +107,6 @@ describe('profiling related functions', function()
     return tonumber(s) + tonumber(us) / 1000000
   end
 
-  describe('profile_equal', function()
-    itp('times are equal to themselves', function()
-      local start = profile_start()
-      assert.is_true(profile_equal(start, start))
-
-      local e = profile_end(start)
-      assert.is_true(profile_equal(e, e))
-    end)
-
-    itp('times are unequal to others', function()
-      assert.is_false(profile_equal(profile_start(), profile_start()))
-    end)
-  end)
-
```

Comment:
These tests seem still useful. Perhaps it's better to keep `profile_equal()` as non-static instead?

Reply:
No strong opinion. 

```
/// Checks if time `tm1` is equal to `tm2`.
///
/// @return true if `tm1` == `tm2`
static bool profile_equal(proftime_T tm1, proftime_T tm2) FUNC_ATTR_CONST
{
  return tm1 == tm2;
}
```

if that function was more complicated, i may reconsider.
Basic equality check is enough to shim `profile_equal()` outside C codebase.